### PR TITLE
Vscode graphql updates

### PR DESCRIPTION
**Pull request by willdch**
Added a `.graphqlrc.yaml` file needed for the recommended Graphql extensions to give better Graphql `syntax highlighting`, `auto-completion` and `go to definition` support.

### CODE REVIEW

**Review by CarsonF — COMMENTED** [review]
Extrapolate review requests from here please
https://github.com/SeedCompany/seed-api/pull/428

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 PORT=3001
+CORD_URL=https://api.dev.cordfield.com
```
Comment (CarsonF) [inline .env:2]: Don't add another env var for the same purpose. The one below this is fine. 

```suggestion
```

File: .graphqlrc.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+schema: schema.graphql
+extensions:
+  endpoints:
+    default:
+      url: ${CORD_URL}/graphql
+documents: src/**/*.graphql
```
Comment (CarsonF) [inline .graphqlrc.yaml:6]:
```suggestion
documents: src/**/*.graphql

```

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Comment by sethmcknight** [comment]
This lived to the end of its lifetime and died

**Pull request closed without merging**